src/app.js

```diff
@@ -1 +1,42 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { generateRandomNumber } = require('./generateRandomNumber');
+const { isValid } = require('./isInputValid');
+const { terminal } = require('./terminal');
+const { calculateBullsAndCows } = require('./calculateCowsAndBulls');
+
+const generetadNum = generateRandomNumber();
+
+const checkUserAnswer = (answer) => {
+  if (!isValid) {
+    console.log('Insert a number of 4 different digits');
+
+    terminal.question(
```

Comment:
You have duplicated code here and in 36–39 lines. You can use `playGame()` here to avoid duplicates.